README.md

```diff
@@ -14,7 +14,7 @@ This is a list of clouds that we test on a nightly basis against our `scs-compat
 | [Wavestack](https://www.noris.de/wavestack-cloud/)                                                             | Public cloud for customers                        | noris network AG/Wavecon GmbH | [![Compliance Status](https://img.shields.io/github/actions/workflow/status/SovereignCloudStack/standards/check-wavestack-v3.yml?label=v3)](https://github.com/SovereignCloudStack/standards/actions/workflows/check-wavestack-v3.yml) |                               [HM](https://health.wavestack1.sovereignit.cloud:3000/)                                    |
 | [REGIO.cloud](https://regio.digital)                                                                           | Public cloud for customers                        | OSISM GmbH                    | [![Compliance Status](https://img.shields.io/github/actions/workflow/status/SovereignCloudStack/standards/check-regio-a-v4.yml?label=v4)](https://github.com/SovereignCloudStack/standards/actions/workflows/check-regio-a-v4.yml)   |   broken <!--[HM](https://apimon.services.regio.digital/public-dashboards/17cf094a47404398a5b8e35a4a3968d4?orgId=1&refresh=5m)-->      |
 | [CNDS](https://cnds.io/)                                                                                       | Public cloud for customers                        | [artcodix UG](https://artcodix.com/) | [![Compliance Status](https://img.shields.io/github/actions/workflow/status/SovereignCloudStack/standards/check-artcodix-v3.yml?label=v3)](https://github.com/SovereignCloudStack/standards/actions/workflows/check-artcodix-v3.yml)  |                                 [HM](https://ohm.muc.cloud.cnds.io/)                                              |
-| [aov.cloud](https://aov.de/)                                                                                   | Community cloud for customers                     | aov IT.Services GmbH          |    (soon)                                                                                                                                             |                               [HM](https://health.aov.cloud/)                                                            |
+| [aov.cloud](https://www.aov.de/)                                                                                   | Community cloud for customers                     | aov IT.Services GmbH          |    (soon)                                                                                                                                             |                               [HM](https://health.aov.cloud/)                                                            |
```

Comment:
```suggestion
| [aov.cloud](https://www.aov.de/)                                                                               | Community cloud for customers                     | aov IT.Services GmbH          |    (soon)                                                                                                                                             |                               [HM](https://health.aov.cloud/)                                                            |
```

Reply:
In this suggested change, I removed 4 spaces to make up for the 4 new characters `www.`